# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1246277831 -7200
#      Mon Jun 29 14:17:11 2009 +0200
# Branch release-3-2-x
# Node ID 1d6856f0acd3cd4fa0231dd6e3e593c0e144eace
# Parent  9d85d65615731b1ffe94926f56846dd31c264284
Add exported symbols

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,13 @@
+2009-06-26  Michael Goffioul  <michael.goffioul@gmail.com>
+
+	* pathsearch.h (class dir_path::static_members): Decorate with
+	OCTAVE_API.
+
 2009-06-24  Alexander Barth  <barth.alexander@gmail.com>
 
 	* eigs-base.cc (EigsRealSymmetricMatrix,
 	EigsRealSymmetricMatrixShift, EigsRealSymmetricFunc,
 	EigsRealNonSymmetricMatrix, EigsRealNonSymmetricMatrixShift,
 	EigsRealNonSymmetricFunc, EigsComplexNonSymmetricMatrix,
 	EigsComplexNonSymmetricMatrixShift, EigsComplexNonSymmetricFunc):
 	Use octave_idx_type for parameters of type LOGICAL in ARPACK.
diff --git a/liboctave/pathsearch.h b/liboctave/pathsearch.h
--- a/liboctave/pathsearch.h
+++ b/liboctave/pathsearch.h
@@ -120,17 +120,17 @@ private:
   string_vector pv;
 
   void init (void);
 
   // Use a singleton class for these data members instead of just
   // making them static members of the dir_path class so that we can
   // ensure proper initialization.
 
-  class static_members
+  class OCTAVE_API static_members
   {
   public:
 
     static_members (void);
 
     static char path_sep_char (void)
     {
       return instance_ok () ? instance->xpath_sep_char : 0;
